src/TimeSteppers/quasi_adams_bashforth_2.jl

```diff
@@ -37,10 +36,10 @@ timestep (`G⁻`).
     Uⁿ⁺¹ = Uⁿ + Δt * Gⁿ
     ```
 """
-function QuasiAdamsBashforth2TimeStepper(grid, tracers, χ = 0.1;
+function QuasiAdamsBashforth2TimeStepper(grid, prognostic_fields, χ = 0.1;
                                          implicit_solver::IT = nothing,
-                                         Gⁿ = TendencyFields(grid, tracers),
-                                         G⁻ = TendencyFields(grid, tracers)) where IT
+                                         Gⁿ = deepcopy(prognostic_fields),
+                                         G⁻ = deepcopy(prognostic_fields)) where IT
```

Comment:
should we use `similar` mapped over the named tuple instead? The boundary conditions can contain arrays, which may be unnecessary to copy.